requirements/dev-requirements.txt

```diff
@@ -24,6 +24,8 @@ babel==2.9.1
     # via sphinx
 backcall==0.2.0
     # via ipython
+beaker==1.11.0
```

Comment:
Is this used to manage sessions for `oic`, or is it just a requirement of the library that doesn't happen to be used in our setup? [Beaker](https://pypi.org/project/Beaker/) is a session library, and I don't think we want another one of those since we are using Django sessions.

Reply:
That doesn't seem to be the case. I actually don't see where `oic` is using it...